astro/src/pages/volunteer.astro

```diff
@@ -0,0 +1,242 @@
+---
+import Layout from '../layouts/Layout.astro';
+import Section from '../components/Section.astro';
+---
+
+<Layout title="Volunteer">
+ <Section theme="white p-4">
+  <h2 class="display-4 fw-bold">Volunteer Teams</h2>
+  <div class="col-md-9">
+    <p class="display-6">Thank you for wanting to learn about our volunteer opportunities at Accessible Community. We have five different teams to choose from, and we have provided decriptions of each team to help you choose one (or more) to join. Everyone, including our staff, is a volunteer.</p>
+    <p class="display-6"><strong>All our teams meet virtually on Zoom.</strong> This allows us to have an international team of volunteers.</p>
```

Comment:
Can you put some sort of button or link to the Volunteer Form, so that people can jump down there immediately?